wca-guidelines.md

```diff
@@ -161,6 +161,7 @@ To be more informative, each Guideline is classified using one of the following
 - A3c4++) [CLARIFICATION] The competitor may remove the object during the solve, if they did not do it during the inspection (e.g. they forgot).
 - A5b+) [CLARIFICATION] While inspecting or solving the puzzle, the competitor may touch the puzzle with any part of their body. Exception: One-Handed Solving and Solving With Feet (see [Regulation C1b](regulations:regulation:C1b) and [Regulation D1b](regulations:regulation:D1b)).
 - A6b+) [EXPLANATION] The arbitrary value of 0.06 seconds was chosen to accommodate concerns about Stackmat timer malfunctions.
+- A6e+) [CLARIFICATION] If the puzzle falls into the lap of the competitor after stopping the timer, this is considered as touching the puzzle.
```

Comment:
I keep trying to convince myself it's not too weird, but for some reason I just feel like the word "as" should be removed.

Reply:
How about a total rephrase? Something like:

> The competitor is considered to be touching the puzzle if ....

Reply:
I like this suggestion, it's changed now!

Reply:
I'm a little concerned about that version, because it can sound like we are trying to define a competitor to be touching the cube as the cube falling into their lap. I strongly prefer the previous version, because clarity is the most important thing.